src/System.IO/tests/TextReader/TextReaderTests.cs

```diff
@@ -0,0 +1,283 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class TextReaderTests
+    {
+        protected virtual char[] GetSmallData()
+        {
+            return new char[] { 'H', 'E', 'L', 'L', 'O' };
+        }
+
+        protected virtual char[] GetLargeData()
+        {
+            char[] testData = new char[] { 'H', 'E', 'L', 'L', 'O' };
+
+            List<char> data = new List<char>();
```

Comment:
nit, up to you, but when assigning the result of a `new` expression, the type is obvious, so we usually use `var` on the left

Reply:
Changed... will follow that now on...